core/src/main/scala/cromwell/core/actor/BatchingDbWriter.scala

```diff
@@ -69,9 +69,13 @@ trait BatchingDbWriterActor { this: FSM[BatchingDbWriterState, BatchingDbWriterD
   
   def isShuttingDown: Boolean = shuttingDown
   def dbFlushRate: FiniteDuration
+  def dbBatchSize: Int
+  def writeActorName: String
```

Comment:
ToL: I think we could just use `self.path.name` which should be the name of the actor. And if it isn't we should set it so when we create it as it shows up in the log and is good practice IMO. Again ToL :)